### DISCUSSION TITLE
Colored cell table

### DISCUSSION THREAD

**Discussion by Dmitriy147** [fyne-io/fyne#4349] [General]
How can I correctly make selectively colored table cells? Unfortunately, when scrolling through a long list with the mouse, the color begins to highlight chaotically on v.2.4.1!

**Comment by andydotxyz** [comment]
You can add colour with a rectangle at the background of your cell.
Remember cells are re-used so you must turn the colour off when not wanted. Perhaps this is what you mean by "chaotic"?

**Comment by Dmitriy147** [comment]
I try to highlight some cells and on the previous version 2.3.5 i was able to solve this task by using canvas.NewText color..
Another variant wich i have try - importance of the lable in table. But no variant is working correct in 2.4.1.
Could you please write a simple example of how to make one cell with background color?

**Reply by andydotxyz** [reply]
Why don't you open a bug for what was working before but no longer works? Every upgrade should be backward compatible.

**Reply by Dmitriy147** [reply]
I am not sure that my solution of the problem on the previous versions was correct!

**Comment by Dmitriy147** [comment]
Bug was opened...
https://github.com/fyne-io/fyne/issues/4351#issue-1967773587
